### PR TITLE
Miscelaneous fixes for ods-ci 1.11.0

### PR DESCRIPTION
**Pull request by jgarciao**

### CODE REVIEW

**Comment by jgarciao** [comment]
It's being tested right now here: https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/918

**Comment by mwaykole** [comment]
I would suggest running smoke/sanity with these changes to avoid risk

**Review by lugi0 — APPROVED** [review]
LGTM but I agree with Milind, especially to confirm that the changes in `tests/Resources/Page/ODH/Prometheus/Prometheus.robot` are fine

File: tests/Tests/100__deploy/130__operators/130__rhods_operator/130__rhods_operator.robot [hunk 1]
```diff
@@ -14,7 +14,7 @@ ${commercial_url}              https://www.redhat.com/en/technologies/cloud-comp
 *** Test Cases ***
 Verify RHODS operator information
   [Documentation]  This TC verfiy if the text present in RHODS opeartor Details section
-  [Tags]  ODS-498   ODS-624   Sanity
+  [Tags]  ODS-498   ODS-624   Sanity    FlakyTest
```
Comment (tarukumar) [inline tests/Tests/100__deploy/130__operators/130__rhods_operator/130__rhods_operator.robot:17]: i would suggest to have tags as defined in guidline.
example:
[Tags]  Sanity
    ...     ODS-498
    ...     ODS-624
    ...     FlakyTest

Comment (jgarciao) [reply tests/Tests/100__deploy/130__operators/130__rhods_operator/130__rhods_operator.robot:17]: As the PR already has one approval, do you mind if I address it in a future PR?

Comment (tarukumar) [reply tests/Tests/100__deploy/130__operators/130__rhods_operator/130__rhods_operator.robot:17]: no worries. I will approve as well

**Review by tarukumar — APPROVED** [review]
LGTM

**Pull request merged**